### PR TITLE
move version and author info outside __init__ to allow proper pip install

### PR DESCRIPTION
**Pull request by ziirish**
Hello,

I am using your awesome lib in a [Octoprint](https://octoprint.org/) plugin called [TuyaSmartplug](https://github.com/ziirish/OctoPrint-TuyaSmartplug).

However, I'm encountering an issue where I can't just `pip install pytuya`.
This is because the setup.py imports *pytuya.__init__* in order to retrieve the `__author__` and `__version__` but the *__init__* also imports `pyaes` which is not yet installed at the time I run `pip install`.

You'll find the corresponding bug report here: https://github.com/ziirish/OctoPrint-TuyaSmartplug/issues/6

This patch attempts to resolve this situation.

### CODE REVIEW

File: pytuya/__init__.py [hunk 1]
```diff
@@ -27,16 +27,14 @@
     Crypto = AES = None
     import pyaes  # https://github.com/ricmoo/pyaes
 
+from pytuya.const import __version__
```
Comment (clach04) [inline pytuya/__init__.py:30]: Looks like this file is missing from the PR/commit

File: pytuya/__init__.py [hunk 2]
```diff
@@ -252,8 +250,8 @@ def generate_payload(self, command, data=None):
         #print('postfix_payload %r' % hex(len(postfix_payload)))
         assert len(postfix_payload) <= 0xff
         postfix_payload_hex_len = '%x' % len(postfix_payload)  # TODO this assumes a single byte 0-255 (0x00-0xff)
-        buffer = hex2bin( payload_dict[self.dev_type]['prefix'] + 
-                          payload_dict[self.dev_type][command]['hexByte'] + 
+        buffer = hex2bin( payload_dict[self.dev_type]['prefix'] +
```
Comment (clach04) [inline pytuya/__init__.py:253]: removing trailing space is great - ideally should be a separate commit. Not the end of the world, will likely accept :)

**Review by clach04 — CHANGES_REQUESTED** [review]
Apologies to not seeing this sooner. Thanks for taking the time to work on this along with a fix. Looks good, but based on review I think a file is missing. Happy to approve once it if you can add it.

File: pytuya/__init__.py (hunk 1 above)
Comment (ziirish) [reply pytuya/__init__.py:30]: Oh sorry, missing `git add`... This is now fixed.

File: pytuya/__init__.py (hunk 2 above)
Comment (ziirish) [reply pytuya/__init__.py:253]: Yes sorry, I noticed this afterwards as well. Must be my vim config that strips trailing spaces automagically.

File: pytuya/__init__.py (hunk 1 above)
Comment (clach04) [reply pytuya/__init__.py:30]: That was fast! 👍

**Comment by clach04** [comment]
Thanks @ziirish, good solution! 💯

**Comment by ziirish** [comment]
Thanks for your kind review ;)

**Comment by nicole-ashley** [comment]
Unfortunately I don't think this solution had the effect it was meant to, or at least as much as it should.

The issue I'm seeing is that crypto is required by `__init__`. This fix _appears_ to remove the import of `__init__` by instead importing from `const` instead. However python still processes `__init__` to understand the package when you try to import anything from that folder, and when `Crypto` and `pyaes` aren't available (eg, a fresh install) it fails.

I would suggest the quickest fix is to just hard-code the Author and Version information in `setup.py`. A more in-depth (but breaking) fix would be to move the device classes into their own files to remove the crypto imports from `__init__.py`.

**Comment by ziirish** [comment]
I confirm the fix didn't have the expected impact.
The installation is still broken on python2.
